src/components/Admin/Toolbar.jsx

```diff
@@ -62,7 +64,36 @@ const Toolbar = ({
     );
     setToggle(!toggle);
   };
+  // const csvData = objects.entries(download).map(entry => rowData[entry[0]] = entry[1]);
+  // const csvData = objects;
+  const csvData = objects.map((download) => {
+    const rowData = {};
+
+    Object.entries(download).forEach(([key, value]) => {
+      if (key === "members" && Array.isArray(value) && value.length > 0) {
+        const memberNames = value.map((member) => member.name).join(", ");
+        const memberEmails = value.map((member) => member.email).join(", ");
+
+        rowData.members = `Names: ${memberNames}, Emails: ${memberEmails}`;
+      } else {
+        if (typeof value === "boolean") {
+          rowData[key] = value ? "Yes" : "No";
+        } else {
+          rowData[key] = value || "";
```

Comment:
in what situations does this occur?

Reply:
![image](https://github.com/acm-ucr/hackathon-website/assets/49844738/c885a0ab-a7a2-49a3-9104-5603a4e1e33d)
bool is needed

Reply:
otherwise it is blank

Reply:
the last line is needed. Here is what it looks like when you remove both:
![image](https://github.com/acm-ucr/hackathon-website/assets/49844738/d4c676cf-4190-4ee6-a043-fdbf0bf00837)